src/interface/meta.thrift

```diff
@@ -416,6 +416,46 @@ struct CheckPasswordReq {
     2: string encoded_pwd,
 }
 
+enum ConfigType {
+    INT64   = 0x00,
+    DOUBLE  = 0x01,
+    BOOL    = 0x02,
+    STRING  = 0x03,
+} (cpp.enum_strict)
+
+struct ConfigItem {
+    1: string               space,
```

Comment:
Why not spaceId ?

Reply:
At first I encrypt with spaceId, but later I found I need to get space name when I get or list config. There is only a map of <name, spaceId> in MetaClient, no need to add another one? Or do you have any better ideas?